src/pykeen/trackers.py

```diff
@@ -123,7 +122,7 @@ def __init__(
         if offline:
             os.environ['WANDB_MODE'] = 'dryrun'
 
-        self._wandb_run = self.wandb.init(project=self.project, name=experiment, **kwargs)
+        self.wandb_run = self.wandb.init(project=self.project, name=experiment, **kwargs)
```

Comment:
this name needs to match up to the one in the type annotation. I'd prefer run because the prefix `wandb` is a bit redundant

Reply:
I see. I thought it would reduce the potential that - if there is some future change to the result tracker's base class - that there wouldn't be a conflict. It is, after all, a wandb specific property.

Reply:
My bad for the name mismatch :) sorry
Should I use run or wandb_run then?

Reply:
Go with `run`. If we ever run into collisions, then we will come up with a better solution